### PR TITLE
Very basic set up

### PR DESCRIPTION
**Pull request by svyatoslavryvkov**
This is not working for now, but the main structure laid. Settled basic configuration of whole app.
@texazcowboy fyi

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,21 +1,37 @@
 {
-  "name": "feeder-client",
+  "name": "my-app",
```
Comment (texazcowboy) [inline package.json:2]: lets leave original name

File: src/api/domain.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+let domain: string;
+
+if (process.env.NODE_ENV === 'production') {
+    domain = '/';
+} else {
+    domain = 'https://localhost:8080';
```
Comment (texazcowboy) [inline src/api/domain.ts:6]: maybe we can move base-url to environmental variable or something similar?

File: package.json (hunk 1 above)
Comment (svyatoslavryvkov) [reply package.json:2]: I will return previous

File: src/api/domain.ts (hunk 2 above)
Comment (svyatoslavryvkov) [reply src/api/domain.ts:6]: Yes, sure, it's just for now. There are a lot of tiny things that I have to keep in mind while set up project. As project will grow, we will remove such code-smell stuff

**Review by texazcowboy — APPROVED** [review]

**Pull request closed without merging**